noise_api/noise_analysis/noisemap.py

```diff
@@ -198,7 +198,4 @@ def run_noise_calculation(task_def: dict):
     #   https: // github.com / Ifsttar / NoiseModelling / blob / master / noisemap - core / src / main / java / org /
     #   orbisgis / noisemap / core / jdbc / JdbcNoiseMap.java  # L68
 
-    # if calculation_settings["result_format"] == "png":
-    #     return convert_result_to_png(noise_result_geojson)
-
-    return {"result_format": task_def["result_format"], "geojson": noise_result_geojson}
+    return {"geojson": noise_result_geojson}
```

Comment:
the final response would then look like this right? 
```
{
        "task_id": async_result.id,
        "task_state": async_result.state,
        "task_succeeded": async_result.successful(),
        "result_ready": async_result.ready(),
        "result": {
"geojson": {.........the actual result....}
}
```
 maybe its better to take out the extra {"geojson": {}} level? Do we need the state info from task.state?  I ready about these getters today.
 .ready() returns false while calculation is pending, but true if ended in failure  
.successful() returns True only if successfully finished.
 It might be all the info we need?